hc/accounts/tests/test_check_token.py

```diff
@@ -25,15 +25,26 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     def test_redirects_already_logged_in(self):
-        """test that a logged in user redirects when they log in again""" 
-        #Login and test it redirects already logged in
-        self.client.post(reverse('hc-check-token', args = ["alice", "secret-token"]))
-        re = self.client.post(reverse('hc-check-token', args = ["alice", "secret-token"]))
+        """test that a logged in user redirects when they log in again"""
+        # Login and test it redirects already logged in
```

Comment:
Given that you have already included a docstring on the previous line, I think adding a single line comment would be a duplication.

Reply:
pep8 changed line 29 to that. must have been ###